api.md

```diff
@@ -1296,16 +1296,22 @@ HINT:  Decrease the replication factor or attach more data nodes to the hypertab
 ## show_chunks() [](show_chunks)
 Get list of chunks associated with hypertables.
 
+Function accepts the following required and optional arguments. These arguments 
+have the same semantics as the `drop_chunks` [function](#drop_chunks).
+
+#### Required Arguments [](show_chunks-required-arguments)
+
+|Name|Description|
+|---|---|
+| `relation` | (REGCLASS) Hypertable or continuous aggregate from which to select chunks. If not supplied, all chunks are shown. |
```

Comment:
What does it mean for the argument to be "required", but then "if not supplied, all chunks are shown"?

Reply:
It means that I am blind :) Fixing